### PR TITLE
OP-530 standardize button text and mnemonics.

### PR DESCRIPTION
**Pull request by dbmalkovsky**
This is a fairly complete pass through the code base although minor omissions/changes most likely will be needed overtime.

### CODE REVIEW

File: src/main/java/org/isf/operation/gui/OperationRowEdit.java [hunk 1]
```diff
@@ -313,7 +313,8 @@ public OperationRowEdit(OperationRow opRow) {
 		add(panelButtons, BorderLayout.SOUTH);
 		panelButtons.setLayout(new FlowLayout(FlowLayout.RIGHT, 5, 5));
 		
-		JButton btnSaveButton = new JButton(MessageBundle.getMessage("angal.operationrowedit.save")); //$NON-NLS-1$
+		JButton btnSaveButton = new JButton(MessageBundle.getMessage("angal.common.save.btn"));
+		btnSaveButton.setMnemonic(MessageBundle.getMnemonic("angal.common.save.btn.key"));
 		btnSaveButton.addMouseListener(new MouseAdapter() {
```
Comment (mwithi) [inline src/main/java/org/isf/operation/gui/OperationRowEdit.java:318]: btnSaveButton -> maybe -> just btnSave

File: src/main/java/org/isf/therapy/gui/TherapyEdit.java [hunk 2]
```diff
@@ -413,9 +412,9 @@ private JPanel getVisitPanel() {
 
 	private JButton getRemoveVisitButton() {
 		if (removeVisit == null) {
-			removeVisit = new JButton(MessageBundle.getMessage("angal.therapy.removevisit")); //$NON-NLS-1$
+			removeVisit = new JButton(MessageBundle.getMessage("angal.therapy.removevisit.btn"));
+			removeVisit.setMnemonic(MessageBundle.getMnemonic("angal.therapy.removevisit.btn.key"));
 			removeVisit.setIcon(new ImageIcon("rsc/icons/delete_button.png"));
-			removeVisit.setMnemonic(KeyEvent.VK_W);
```
Comment (mwithi) [inline src/main/java/org/isf/therapy/gui/TherapyEdit.java:418]: maybe removeVisit -> removeVisitButton

File: src/main/java/org/isf/therapy/gui/TherapyEdit.java [hunk 3]
```diff
@@ -467,9 +466,9 @@ public void actionPerformed(ActionEvent e) {
 	
 	private JButton getAddVisitButton() {
 		if (addVisit == null) {
-			addVisit = new JButton(MessageBundle.getMessage("angal.therapy.addvisit")); //$NON-NLS-1$
+			addVisit = new JButton(MessageBundle.getMessage("angal.therapy.addvisit.btn"));
+			addVisit.setMnemonic(MessageBundle.getMnemonic("angal.therapy.addvisit.btn.key"));
 			addVisit.setIcon(new ImageIcon("rsc/icons/calendar_button.png"));
-			addVisit.setMnemonic(KeyEvent.VK_V);
 			addVisit.setMaximumSize(new Dimension(VisitButtonWidth, AllButtonHeight));
 			addVisit.setHorizontalAlignment(SwingConstants.LEFT);
```
Comment (mwithi) [inline src/main/java/org/isf/therapy/gui/TherapyEdit.java:473]: maybe addVisit -> addVisitButton

File: src/main/java/org/isf/therapy/gui/TherapyEdit.java [hunk 4]
```diff
@@ -874,9 +873,9 @@ private JLabel getTherapyCheckLabel() {
 
 	private JButton getCheckTherapyButton() {
 		if (checkTherapy == null) {
-			checkTherapy = new JButton(MessageBundle.getMessage("angal.therapy.checkavailability")); //$NON-NLS-1$
+			checkTherapy = new JButton(MessageBundle.getMessage("angal.therapy.checkavailability.btn"));
+			checkTherapy.setMnemonic(MessageBundle.getMnemonic("angal.therapy.checkavailability.btn.key"));
 			checkTherapy.setIcon(new ImageIcon("rsc/icons/flag_button.png"));
-			checkTherapy.setMnemonic(KeyEvent.VK_C);
```
Comment (mwithi) [inline src/main/java/org/isf/therapy/gui/TherapyEdit.java:879]: checkTerapyButton

File: src/main/java/org/isf/therapy/gui/TherapyEdit.java [hunk 5]
```diff
@@ -952,11 +951,10 @@ protected void updateCheckLabel() {
...
-			addTherapy = new JButton(MessageBundle.getMessage("angal.therapy.addtherapy")); //$NON-NLS-1$
+			addTherapy = new JButton(MessageBundle.getMessage("angal.therapy.addtherapy.btn"));
+			addTherapy.setMnemonic(MessageBundle.getMnemonic("angal.therapy.addtherapy.btn.key"));
 			addTherapy.setIcon(new ImageIcon("rsc/icons/therapy_button.png"));
-			addTherapy.setMnemonic(KeyEvent.VK_A);
-			addTherapy.setMaximumSize(new Dimension(TherapyButtonWidth,
-					AllButtonHeight));
+			addTherapy.setMaximumSize(new Dimension(TherapyButtonWidth,	AllButtonHeight));
 			addTherapy.setHorizontalAlignment(SwingConstants.LEFT);
 			addTherapy.addActionListener(new ActionListener() {
```
Comment (mwithi) [inline src/main/java/org/isf/therapy/gui/TherapyEdit.java:959]: addTherapyButton

File: src/main/java/org/isf/therapy/gui/TherapyEdit.java [hunk 6]
```diff
@@ -1001,9 +999,9 @@ public void actionPerformed(ActionEvent e) {
...
 	private JButton getEditTherapyButton() {
 		if (editTherapy == null) {
-			editTherapy = new JButton(MessageBundle.getMessage("angal.therapy.edittherapy")); //$NON-NLS-1$
+			editTherapy = new JButton(MessageBundle.getMessage("angal.therapy.edittherapy.btn"));
+			editTherapy.setMnemonic(MessageBundle.getMnemonic("angal.therapy.edittherapy.btn.key"));
 			editTherapy.setIcon(new ImageIcon("rsc/icons/therapy_button.png"));
-			editTherapy.setMnemonic(KeyEvent.VK_E);
 			editTherapy.setMaximumSize(new Dimension(TherapyButtonWidth,
 					AllButtonHeight));
 			editTherapy.setHorizontalAlignment(SwingConstants.LEFT);
```
Comment (mwithi) [inline src/main/java/org/isf/therapy/gui/TherapyEdit.java:1007]: editTherapyButton

File: src/main/java/org/isf/therapy/gui/TherapyEdit.java [hunk 7]
```diff
@@ -1051,7 +1049,8 @@ public void actionPerformed(ActionEvent e) {
 	
 	private JButton getRemoveTherapyButton() {
 		if (removeTherapy == null) {
-			removeTherapy = new JButton(MessageBundle.getMessage("angal.therapy.removetherapy")); //$NON-NLS-1$
+			removeTherapy = new JButton(MessageBundle.getMessage("angal.therapy.removetherapy.btn"));
+			removeTherapy.setMnemonic(MessageBundle.getMnemonic("angal.therapy.removetherapy.btn.key"));
 			removeTherapy.setIcon(new ImageIcon("rsc/icons/delete_button.png"));
 			removeTherapy.setMaximumSize(new Dimension(TherapyButtonWidth, AllButtonHeight));
 			removeTherapy.setHorizontalAlignment(SwingConstants.LEFT);
```
Comment (mwithi) [inline src/main/java/org/isf/therapy/gui/TherapyEdit.java:1056]: removeTherapyButton

File: src/main/java/org/isf/utils/jobjects/Cropping.java [hunk 8]
```diff
@@ -132,7 +133,8 @@ public BufferedImage clipImage() {
 	}
 
 	public JPanel getUIPanel() {
-		JButton clip = new JButton("save");
+		JButton clip = new JButton(MessageBundle.getMessage("angal.common.save.btn"));
+		clip.setMnemonic(MessageBundle.getMnemonic("angal.common.save.btn.key"));
 		clip.addActionListener(new ActionListener() {
 			public void actionPerformed(ActionEvent e) {
 				clipImage();
```
Comment (mwithi) [inline src/main/java/org/isf/utils/jobjects/Cropping.java:140]: clipButton

Comment (mwithi) [reply src/main/java/org/isf/utils/jobjects/Cropping.java:140]: or also saveButton

File: src/main/java/org/isf/visits/gui/VisitView.java [hunk 9]
```diff
@@ -436,8 +436,8 @@ public JPanel getVisitDateChooserPanel() {
 
 			dateViPanel = new JPanel();
 
-			dateAdm = new JButton();
-			dateAdm.setText(MessageBundle.getMessage("angal.visit.gotodate") + ":"); //$NON-NLS-1$
+			dateAdm = new JButton(MessageBundle.getMessage("angal.visit.gotodate.btn"));
+			dateAdm.setMnemonic(MessageBundle.getMnemonic("angal.visit.gotodate.btn.key"));
 			dateAdm.addActionListener(new ActionListener() {
```
Comment (mwithi) [inline src/main/java/org/isf/visits/gui/VisitView.java:441]: dateAdmissionButton

**Review by mwithi — APPROVED** [review]
LGTM, just some renaming suggestions. Thanks!

File: src/main/java/org/isf/operation/gui/OperationRowEdit.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/operation/gui/OperationRowEdit.java]: Thanks; done

File: src/main/java/org/isf/therapy/gui/TherapyEdit.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/therapy/gui/TherapyEdit.java]: Thanks; done

File: src/main/java/org/isf/therapy/gui/TherapyEdit.java (hunk 3 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/therapy/gui/TherapyEdit.java]: Thanks; done

File: src/main/java/org/isf/therapy/gui/TherapyEdit.java (hunk 4 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/therapy/gui/TherapyEdit.java]: Thanks; done

File: src/main/java/org/isf/therapy/gui/TherapyEdit.java (hunk 5 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/therapy/gui/TherapyEdit.java]: Thanks; done

File: src/main/java/org/isf/therapy/gui/TherapyEdit.java (hunk 6 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/therapy/gui/TherapyEdit.java]: Thanks; done

File: src/main/java/org/isf/therapy/gui/TherapyEdit.java (hunk 7 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/therapy/gui/TherapyEdit.java]: Thanks; done

File: src/main/java/org/isf/utils/jobjects/Cropping.java (hunk 8 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/utils/jobjects/Cropping.java]: Thanks; done

File: src/main/java/org/isf/visits/gui/VisitView.java (hunk 9 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/visits/gui/VisitView.java]: Thanks; done

**Review by mwithi — APPROVED** [review]
Thanks!

**Pull request merged**